MvvmCross/Droid/Shared/Presenter/MvxFragmentsPresenter.cs

```diff
@@ -67,6 +67,11 @@ protected virtual void ShowFragment(MvxViewModelRequest request)
             var serializedRequest = Serializer.Serializer.SerializeObject(request);
             bundle.PutString(ViewModelRequestBundleKey, serializedRequest);
 
+            if (request is MvxViewModelInstanceRequest)
+            {
+                Mvx.Resolve<IMvxChildViewModelCache>().Cache(((MvxViewModelInstanceRequest)request).ViewModelInstance);
```

Comment:
The result is only cached until the fragment is shown and the result is attached as viewmodel. It doesn't make sense to cache it here because it will never reach this point again.